modules/installation-nutanix-failure-domains-req.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/nutanix-failure-domains.adoc
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-nutanix-failure-domains-req_{context}"]
+= Failure domain requirements
+
+When planning to use failure domains, consider the following requirements:
+
+* All Prism Element instances must be managed by the same instance of Prism Central. A deployment that is comprised of multiple Prism Central instances is not supported.
+* All Prism Element instances must share the same network subnet. Only one subnet per {product-title} cluster is supported.
```

Comment:
Currently only one subnet per Failure Domain is supported. And all the Failure Domain subnets should share the same CIDR address that should also cover the OCP cluster's api and ingress virtual IPs.

Reply:
@yanhua121 I'm working with Mike on this and looking for some clarification on this comment. Let's say that I have one failure domain with subnet 192.168.1.0/24 and a second failure domain with subnet 192.168.2.0/24. Are you saying that the OCP cluster's API and ingress VIPs need to be in one of these two subnets? If not, can you give an example of what you mean by sharing the same CIDR address as the VIPs? thank you

Reply:
In your example, both failure domains' subnets CIDR scope should be 192.168.0.0/16. And the OCP cluster's API and ingress VIP need to be in CIDR 192.168.0.0/16.

Reply:
I'm interpreting "CIDR scope" to mean "both networks are subnets of the same larger network". Is that correct or does "CIDR scope" have a specific meaning here? For example, you couldn't have a failure domain with subnet 192.168.1.0/24 and a second failure domain with subnet 10.0.0.1/24 because they don't share a CIDR scope. Thank you

Reply:
```suggestion
* All Prism Central instances must belong to the same network. You can use one subnet per Failure Domain. For example, you can use a subnet of `192.168.1.0/24` for one Failure Domain and a subnet of `192.168.2.0/24` for another Failure Domain because they both belong to the `192.168.0.0/16` network. The API and Ingress virtual IPs must belong to the network that contains the Failure Domains' subnets.
```
@yanhua121 Is this an accurate summary of your feedback?

Reply:
> I'm interpreting "CIDR scope" to mean "both networks are subnets of the same larger network". Is that correct or does "CIDR scope" have a specific meaning here? For example, you couldn't have a failure domain with subnet 192.168.1.0/24 and a second failure domain with subnet 10.0.0.1/24 because they don't share a CIDR scope. Thank you

@bscott-rh Each failure domain specifies a Prism Element (PE) and the referenced subnet(s) also being defined in this PE. All the subnets referenced in the failure domains defined in the cluster should share the same CIDR scope, with each subnet using a different range of IP addresses of the CIDR scope. And the virtual IP addresses the cluster will use (apiVIP and ingressVIP) should also in this CIDR scope.